### DISCUSSION TITLE
Disable line buttons as dropdown if only 1 button is available

### DISCUSSION THREAD

**Discussion by susanu** [Laravel-Backpack/community-forum#1083] [Ideas]
Hi guys,

I really like the line dropdown with all action buttons but would be cool to disable it if only 1 button is shown.

Thank you!

**Comment by jcastroa87** [comment]
Hello @susanu 

You can set the behaivor of line buttons as [our docs](https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries#merging-line-buttons-into-a-dropdown) says.

You can set it as global or for any controller.

Cheers.

**Reply by susanu** [reply]
Hi @jcastroa87,

I know i can do that, i just suggested that would be cool to automatically disable if only 1 button is visible.

Cheers

**Comment by pxpm** [comment]
Hey @susanu thanks for the suggestion. 

I think I get the idea, having like a "threshold" for displaying the buttons as dropdown. 
Like only show as dropdown if there are 2 or more items. 

I think it's an interesting use-case. I've also thought about other scenario, that would be to have some buttons that never go to the dropdown, or you want to always display 2 buttons and any remaining in a dropdown. 

I don't think we are going to invest time on this for the time being, but I would happily review a PR in case it doesn't make a total mess of this feature. 

Talking specifically about what you required, most of the work is done in https://github.com/Laravel-Backpack/CRUD/blob/6141ed3ffdd3a828fa7a07a0183c7daac98c8aec/src/resources/views/crud/inc/datatables_logic.blade.php#L405

I think we can introduce a new operation setting like: `lineButtonsDropdownThreshold = 1` (so by default we keep the same behavior). 

Let me know if you need my help with something 🙏 

Cheers

**Comment by susanu** [comment]
Hi @pxpm,

Just created a pull request here: https://github.com/Laravel-Backpack/CRUD/pull/5568

Thank you!

**Comment by jcastroa87** [comment]
Great, thanks a lot. We will continue conversation there.

Cheers
